src/modules/Providers/views/migrate/reducer/actions.ts

```diff
@@ -28,7 +28,7 @@ export const SET_NAME = 'SET_NAME';
 export const SET_PROJECT_NAME = 'SET_PROJECT_NAME';
 export const SET_SOURCE_PROVIDER = 'SET_SOURCE_PROVIDER';
 export const SET_SELECTED_VMS = 'SET_SELECTED_VMS';
-const SET_DESCRIPTION = 'SET_DESCRIPTION';
+type SET_DESCRIPTION = 'SET_DESCRIPTION';
```

Comment:
?

Reply:
if the const is only used as a type, TS will throw an error